securedrop/sass/_base.sass

```diff
@@ -322,6 +334,15 @@
       i
         color: #D62727
 
+    &.important
+      border: 1px solid #EBDCEB
+      background: #FDFAFD
+
+      strong
+        color: #673466
+      p
+        color: #555555
```

Comment:
It's probably best to store these hex color codes as variables in  `_variables.sass`